portfolio.html

```diff
@@ -82,5 +82,237 @@ <h2>I'm Amrendra Glad to see you!</h2>
         </ul>
       </div>
     </section>
+    <section class="works">
+      <div class="card">
+        <img class="cardimage" src="./blog_drbl_4x.png" alt="blogimage" />
+        <div class="cardheading">Tonic</div>
+        <div class="canopyitemwrap">
+          <div class="cardcanopy">CANOPY</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">Back End Dev</div>
+          <div class="dot cardlistitem">&bull;</div>
+          <div class="cardlistitem">2015</div>
+        </div>
+        <div class="cardabout">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
```

Comment:
- [ ] Please, consider that the paragraph is a `p` tag, as required in the project.